trino-connector/src/main/java/com/datastrato/gravitino/trino/connector/system/table/GravitinoSystemTableCatalog.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.trino.connector.system.table;
+
+import static io.trino.spi.block.MapValueBuilder.buildMapValue;
+import static io.trino.spi.type.VarcharType.VARCHAR;
+
+import com.datastrato.gravitino.trino.connector.catalog.CatalogConnectorManager;
+import com.datastrato.gravitino.trino.connector.metadata.GravitinoCatalog;
+import com.google.common.base.Preconditions;
+import io.trino.spi.Page;
+import io.trino.spi.block.Block;
+import io.trino.spi.block.BlockBuilder;
+import io.trino.spi.block.MapBlockBuilder;
+import io.trino.spi.connector.ColumnMetadata;
+import io.trino.spi.connector.ConnectorTableMetadata;
+import io.trino.spi.connector.SchemaTableName;
+import io.trino.spi.type.MapType;
+import io.trino.spi.type.TypeOperators;
+import java.util.List;
+
+/** An implementation of the catalog system table * */
+public class GravitinoSystemTableCatalog extends GravitinoSystemTable {
+
+  public static final SchemaTableName TABLE_NAME =
+      new SchemaTableName(SYSTEM_TABLE_SCHEMA_NAME, "catalog");
```

Comment:
Is the name of the system catalog `catalog`? I would suggest using `default`, `catalog` seems to be very strange.

Reply:
Actually, the table name is `catalog`. The meaning  of `default` is not clear.
Users usually query with the table like this:
```
select * from graviton.system.catalog
```

Reply:
So the schema name is `system` and table name is `catalog`?